exercises/parallel-letter-frequency/.meta/src/reference/java/ParallelLetterFrequency.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class ParallelLetterFrequency {
```

Comment:
This (and the public methods) could be package private :)

Reply:
Done!